plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/WebProjectUtil.java

```diff
@@ -83,12 +132,33 @@ public static IFolder getWebInfDirectory(IProject project) {
    * @return the file location or {@code null} if not found
```

Comment:
Can you update the Javadoc to explicitly say

"Note that this can return a file outside the project source directory; it can return a file in a build directory (for example, under `target/m2e-wtp/web-resources/WEB-INF`), which can get cleaned frequently."

I can imagine a developer tries to use this function to find and manipulate `web.xml` in the source repo, not knowing that it can return `web.xml` in a build directory. Also, in this regard, please update the Javadoc to explain this subtle difference from what can happen with `createWebInfFile()`.